### DISCUSSION TITLE
Suggestions for a cheap, reliable, easy to setup wear OS watch?

### DISCUSSION THREAD

**Discussion by crdphone10** [NightscoutFoundation/xDrip#3769] [General]
My son lost his old watch, and we're looking to get him a new one. His phone is a Galaxy s20 FE, which is limited to Android 13. There is a ton of technical detail here on Wear OS connectivity, but not a lot of general feedback. Are there any models that are particularly easy to setup? From reading around, it seems like OS5 is still best avoided due to the new wear watchface standards. I was considering the Galaxy Watch 4 Classic, but I am open to suggestions. He is not picky about features, and we don't need to connect directly to the watch. Any info is appreciated.

**Comment by PhilTU** [comment]
Gw4 would be good. Wear Installer 2 to get the apk installed and select watch face.

**Comment by Diabeticliberty** [comment]
Galaxy Watch 4 and Watch 5 are both good. The 5 has a far longer battery life between charges. You could also take a look at the Mobvoi - Ticwatch series. Very good build and extremely long life between battery charges. Mobvoi sometimes throw in some extra stuff as they have a lot of offers on. Personally as you are using a Samsung phone I would go with the Samsung watch but then I love Samsung stuff. Whichever way you go you will need to install Xdrip+ on the watch using Wear Installer 2 from Malcolm Bryant. The application for Wear Installer 2 is on the Galaxy Play Store. Installation is pretty straight forward.

**Reply by tshakah** [reply]
Which are the best (if any) for acting as collectors so they can work without a phone with a G6?

**Reply by PhilTU** [reply]
https://bluejay.website/ has active development

**Reply by tshakah** [reply]
Thanks for the response. Sadly the GTS isn't available and the U2 isn't waterproof at all, so wouldn't work for me

**Comment by Diabeticliberty** [comment]
I remember reading some time ago that there was a Sony watch that would act as a standalone collector. I don't know if they are still produced but if so I believe that they are waterproof. They are now a very old model and will be very light on features.  If no longer produced you could possibly try the second hand market. Ebay or maybe some other. I would suggest investigating further to make sure I am not speaking out of my rear end. This is something for which I am quite well.kmown unfortunately.

**Reply by PhilTU** [reply]
SONY SW3

You are correct

**Reply by tshakah** [reply]
Thanks. I've bought one second hand to see if it still works

**Reply by AEI-SXSmirror** [reply]
SW3, aka SWR50, can still be had second- and third-hand. Battery life about one day, WearOS 1.5. I have a few of them, and even discovered they would switch to collector mode after half an hour if I happen to forget my phone.
AFAIR they need to be set up with an old version of AndroidWear (I keep a copy of 2.9.0), but they'd even work without a G account ;)
